### PR TITLE
Added components for Airtable integration

### PR DESCRIPTION
**Pull request by smrohrer**
Introduced code components for storing languages and crawled items in an Airtable repository.

The following operations are supported:

LanguageDataStore:
* Retrieve a language by its ISO code
* Retrieve multiple languages by ISO code
* List languages in the data store

ItemDataStore:
* Retrieve/check for existence of an item by its url and the ISO code of the associated language
* Add a crawled item to the data store

When the Wikipedia spider is run, it now queries the data store to get the Wikipedia url's of the given languages. Instead of taking the language objects as an argument, it now takes a request object with a list of ISO codes, and an instance of the data store interface.

### CODE REVIEW

File: wikitongues/wikitongues/data_store/airtable/airtable_item_id_provider.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class AirtableItemIdProvider:
```
Comment (madamc) [inline wikitongues/wikitongues/data_store/airtable/airtable_item_id_provider.py:1]: I'm not sure I understand what role the provider plays. Is it purely a json formatter class?

**Comment by smrohrer** [comment]
My thought was that we use a formula in Airtable (similar to Excel) to assign identifiers, so I’d write a Python class to form identifiers according to the formula in order to query existing items. That way if we ever decide to change how our Airtable works, it is easier to change the code accordingly

File: wikitongues/wikitongues/data_store/airtable/airtable_item_id_provider.py (hunk 1 above)
Comment (smrohrer) [reply wikitongues/wikitongues/data_store/airtable/airtable_item_id_provider.py:1]: My thought was that we use a formula in Airtable (similar to Excel) to assign identifiers, so I’d write a Python class to form identifiers according to the formula in order to query existing items. That way if we ever decide to change how our Airtable works, it is easier to change the code accordingly

Comment (madamc) [reply wikitongues/wikitongues/data_store/airtable/airtable_item_id_provider.py:1]: Ok, so if the identifier is something coming from Airtable and not defined here, then that makes more sense of why it's a provider. Sounds good!

**Pull request merged**